core-tests/jvm/src/test/scala/zio/system/SystemSpec.scala

```diff
@@ -1,44 +1,38 @@
 package zio
 package system
 
-import org.specs2.Specification
-import scala.reflect.io.File
-
-class SystemSpec extends Specification with DefaultRuntime {
-  def is = s2"""
-    Fetch an environment variable and check that:
-      If it exists, return a reasonable value                         $env1
-      If it does not exist, return None                               $env2
-
-    Fetch a VM property and check that:
-      If it exists, return a reasonable value                         $prop1
-      If it does not exist, return None                               $prop2
-
-    Fetch the system's line separator and check that:
-      It is identical to System.lineSeparator                         $lineSep1
-  """
+import zio.test._
+import zio.test.Assertion._
 
-  def env1 = {
-    val io = unsafeRun(system.env("PATH"))
-    io must beSome
-    io.get must contain(File.separator + "bin")
-  }
-
-  def env2 = {
-    val io = unsafeRun(system.env("QWERTY"))
-    io must beNone
-  }
-
-  def prop1 = {
-    val io = unsafeRun(property("java.vm.name"))
-    io must beSome
-    io.get must contain("VM")
-  }
-
-  def prop2 = {
-    val io = unsafeRun(property("qwerty"))
-    io must beNone
-  }
+import scala.reflect.io.File
 
-  def lineSep1 = unsafeRun(lineSeparator) must_=== (java.lang.System.lineSeparator)
-}
+class SystemSpec
+    extends ZIOBaseSpec(
+      suite("SystemSpec")(
+        testM("Fetch an environment variable and check that If it exists, return a reasonable value") {
```

Comment:
You can use nested suites to recover the same structure that the original spec had. See `ZManagedSpec` for an example of this.

```scala
suite("Fetch an environment variable and check that:")(
  testM("If it exists, returns a reasonable value") { ... }
  testM("If it does not exist, return None") { ... }
)
```